addons/dark-www/experimental_scratchr2.css

```diff
@@ -704,3 +704,11 @@ input.link.black {
 .iframeshim {
   color-scheme: light;
 }
+
+/* Forum errors */
+
+.errorlist {
+  background-color: var(--darkWww-box) !important;
+  border-color: var(--darkWww-page-forumErrorColor) !important;
+  color: var(--darkWww-page-forumErrorColor) !important;
```

Comment:
Have you tried without `!important` and if it doesn't work do all 3 need it?

Reply:
Yes, they all need it.

Reply:
Why?

Reply:
dark-www's styles end up below Scratch's styles here.
![The styles getting applied with !important](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/dd241db3-74aa-4498-947d-3954961e93fa)
![The styles not getting applied without !important](https://github.com/ScratchAddons/ScratchAddons/assets/130385691/dafe057d-f603-4a5e-9dbd-86250949cfb7)